sample/ios/fastlane/Fastfile

```diff
@@ -138,7 +138,7 @@ platform :ios do
     end
   end
 
-  desc 'set build number with number of commits'
+  desc 'set build number with the run number'
```

Comment:
@nmint8m We still use `number_of_commits` this case 🤔

Reply:
@doannimble @nmint8m the sample project reflects the core template project. It will be generated automatically. Please ignore the sample.